backend/memory/memory.go

```diff
@@ -376,6 +377,7 @@ func (r *Backend) maintenance() {
 	for range c {
 		fmt.Printf("maintenance: start\n")
 		r.cleanData()
+		r.checkAlerts()
```

Comment:
This should be in the alerts thingsy

Reply:
meaning `alert.go`?

Reply:
what ever you think is best to call it :-)
but in the alert directory, under alerts namespace